test_settings.py

```diff
@@ -15,6 +15,13 @@
         'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
         'LOCATION': 'test-connection-errors',
     },
+    'connection-errors-redis': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'redis://localhost:6379/0',
```

Comment:
Is this guaranteed to actually fail to create a connection on Travis? What would happen to the test if Redis was available?

Reply:
If redis is available then the test would pass even without this fix.

Reply:
you can use `override_settings` to override `CACHES` as well in the test, rather than add yet another cache here?

also the test could assert that this redis cache is inaccessible before executing the check on the ratelimit code

Reply:
Is there a good port for testing, that's valid but guaranteed to have nothing listening on it, like `1` or something? Or finding an open port?

Reply:
I'm using the same string now as the memcache connection tests.

As for the override_settings, I don't know if it works for nested attributes, like changing the "LOCATION" attribute of a specific cache to use the Redis driver. But my branch is open to PRs. :)